apps/files/lib/Listener/SyncLivePhotosListener.php

```diff
@@ -193,4 +184,34 @@ private function handleDeletion(BeforeNodeDeletedEvent $event, Node $peerFile):
 		}
 		return;
 	}
+
+	/*
+	 * Recursively get all the peer ids of a live photo.
+	 * Needed when coping a folder.
+	 */
+	private function handleCopyRecursive(Event $event, Node $sourceNode, Node $targetNode): void {
+		if ($sourceNode instanceof Folder && $targetNode instanceof Folder) {
+			foreach ($sourceNode->getDirectoryListing() as $sourceChild) {
+				$targetNode = $targetNode->get($sourceChild->getName());
+				$this->handleCopyRecursive($event, $sourceChild, $targetNode);
```

Comment:
```suggestion
				$targetChild = $targetNode->get($sourceChild->getName());
				$this->handleCopyRecursive($event, $sourceChild, $targetChild);
```

Otherwise, it fails for the second file.